app/controllers/GoodsRecordsController.scala

```diff
@@ -102,25 +102,29 @@ class GoodsRecordsController @Inject() (
 
   def getDownloadLinkMessagesKey(opt: Option[DownloadDataSummary]): String =
     opt match {
-      case Some(downloadDataSummary) if downloadDataSummary.status == RequestFile    =>
+      case Some(downloadDataSummary) if downloadDataSummary.status == RequestFile     =>
```

Comment:
Don't need to do this, but perhaps this could be written slightly simpler like

opt.map(_.status) match {
case Some(FileInProgress) => 
case Some(FileReadyUnseen) => 
case Some(FileReadySeen) => 
case _ 
}